source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/row/index.tsx

```diff
@@ -92,8 +92,10 @@ class Row extends React.Component<IProps & IProvidedProps> {
 }
 
 export default compose(
-  withRelationship('organization'),
```

Comment:
withRelationship was too rigid to be useful.

Reply:
don't we have a withRelationship in @data/containers that does the same?

Reply:
yeah, it has a todo mentioned in it though